src/Xamarin.Android.Build.Tasks/Tasks/Aot.cs

```diff
@@ -366,15 +377,9 @@ static int GetNdkApiLevel(string androidNdkPath, string androidApiLevel, Android
 			
 		bool RunAotCompiler (string assembliesPath, string aotCompiler, string aotOptions, string assembly)
 		{
-			var arguments = aotOptions + " " + assembly;
-
-			Log.LogMessage (MessageImportance.High, "[AOT] " + assembly);
-			Log.LogMessage (MessageImportance.Low, "Mono arguments: " + arguments);
-			Log.LogMessage (MessageImportance.Low, "MONO_PATH=" + assembliesPath);
-
 			var psi = new ProcessStartInfo () {
 				FileName = aotCompiler,
-				Arguments = arguments,
+				Arguments = aotOptions + " " + assembly,
```

Comment:
`assembly` should probably be quoted, unless the `RunAotCompiler()` caller is quoting it (which would be *weird*), thus:

    Arguments = $"{aotOptions} \"{assembly\"",